bedrock/firefox/templates/firefox/nightly_whatsnew.html

```diff
@@ -39,7 +39,7 @@
           <h2>{{ self.page_title() }}</h2>
           <p class="tagline">
           {% trans %}
-            Every 6 to 8 weeks, a new major version of Firefox is released and as a result, the Nightly version
+            Every 4 to 5 weeks, a new major version of Firefox is released and as a result, the Nightly version
```

Comment:
This page is localized so we should use `l10n_has_tag` for this string change.

Reply:
done